source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -368,4 +373,10 @@ Connect Battery
 
 Connect the battery to the robot side of the Anderson connector. Power on the robot by moving the lever on the top of the 120A main breaker into the ridge on the top of the housing.
 
-If stuff blinks, you probably did it right. From here, you should connect to the roboRIO and try uploading your code!
+If stuff blinks, you probably did it right.
+
```

Comment:
I think it should call about that magenta is for brushless and cyan is for brushed (assuming that is correct)

Reply:
magenta/cyan are brushless (brake and coast), blue/yellow are brushed. I'll tweak the text a bit to better indicate what's what.